lib/nimble_strftime.ex

```diff
@@ -34,7 +34,7 @@ defmodule NimbleStrftime do
   B      | Full month name                                                         | January
   c      | Preferred date+time representation                                      | 2018-10-17 12:34:56
   d      | Day of the month                                                        | 01, 12
-  f      | Microseconds                                                            | 000000, 999999, 0123
+  f      | Microseconds**(this format ignores width and padding options)**             | 000000, 999999, 0123
```

Comment:
Bold is probably too much as it could attention to this particular line. Maybe:

```suggestion
  f      | Microseconds *(does not support width and padding options)*             | 000000, 999999, 0123
```